frontend/src/api/config.ts

```diff
@@ -0,0 +1,19 @@
+import { defaultConfig } from 'swr/_internal';
+import { fetcher, makeSwrHook } from '../api';
+import { SWRConfiguration } from 'swr';
+
+export const getConfig = fetcher.path('/api/v1/config/').method('get').create();
+
+const useGetConfigWithRetry = makeSwrHook('getConfig', getConfig);
+
+export const useGetConfig = (
```

Comment:
Maybe we want to make this a `makeSwrHook` style wrapper, now that there are two "fetchers" that want to work that way?

Reply:
-> #291